src/views/GameView.vue

```diff
@@ -59,7 +59,7 @@
               <div id="opponent-hand-cards" class="d-flex justify-center align-start">
                 <transition name="slide-below" mode="out-in">
                   <transition-group
-                    v-if="hasGlassesEight"
+                    v-if="revealOpponentHand"
```

Comment:
```suggestion
                    v-if="showOpponentHand"
```
Reveal is kinda weird, I like show for the simplicity but I'm open to whatever.